src/heroku/orb.yml

```diff
@@ -25,7 +25,7 @@ commands:
             else
               echo "Heroku is already installed. No operation was performed."
             fi
-  deploy:
+  deploy_to_heroku_via_git:
```

Comment:
What do you think about `deploy-via-git` since we're in the heroku orb already?

Reply:
Thanks @bradylill - made that change